src/less/details/base/details.less

```diff
@@ -0,0 +1,66 @@
+@import "../../less/mixins-common.less";
+
+details.details[open] {
+    .details__icon {
+        transform: rotate(180deg);
+    }
+
+    summary.details__summary::before {
+        content: none;
+    }
+}
+
+/* Remove details marker for non-webkit */
+summary.details__summary::-webkit-details-marker {
```

Comment:
Put this under the next section
summary.details__summary {
    Other styles....
    &::-webkit-details-marker {}
}

Reply:
fixed